### PR TITLE
Readme: update dictionaries and regex sections

### PR DESCRIPTION
**Pull request by artjomsR**
A couple of small things that I spotted. Please feel free to amend / delete sections that you think aren't relevant.

I've also thought about git-adding the binary file for the go proxy in the repo. This would replace the 1st two steps in the One-click mining flow section (we can leave the original instructions below if someone still wants to compile it themselves). But the disadvantage is that these should be regenerated if the script contents changes, and it's difficult to provide these for all 3 OSes (not sure if you'd want PRs for these as it's kinda a security risk?). Still, if you have a Mac and you include the Mac binary, then you save Mac users a bit of hassle :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -182,7 +182,12 @@ You can replace filtered content similarly by entering a string into the "Subtit
 Useful examples of regular expressions:
 
 -   `([(（]([^()（）]|(([(（][^()（）]+[)）])))+[)）])` : Remove names enclosed by parenthesis to indicate speakers (i.e. "**（山田）**　元気ですか？")
--   `\[.*\]` : Remove indications enclosed by brackets that sound or music that is playing (i.e. "**\[PLAYFUL MUSIC]**")
+-   `(.*)\n+(?!-)(.*)` : Some subtitles are split in several lines and this regex forces them into a single line. For this filter to work, you must also put `$1 $2` in the "Subtitle regex filter text replacement" field.
+-   `-?\[.*\]` : Remove indications enclosed by square brackets that sound or music that is playing (i.e. "**\[PLAYFUL MUSIC]**" or "**\-[GASPS]**")
+    - `^[\(\)\.\sA-ZAÂÃÀÇÉÊÍÓÔÕÚÑ]+$` : As an alternative to the above, filter out descriptions written in capital letters but without the square brackets, you descriptions that appear in capital letters. If your language has additional letters with diacritics, you feel free to add them to this list
```
Comment (killergerbah) [inline README.md]: > you descriptions that appear in capital letters

Typo? I can't make out what this is supposed to say.

File: README.md [hunk 2]
```diff
@@ -250,6 +255,8 @@ asbplayer can be setup to support one-click mining workflows by integrating with
 5. Configure Yomitan to use the same note type you have configured for asbplayer.
 6. Using Yomitan's `+` button on asbplayer subtitles will now trigger the flashcard creator with word and definition fields pre-populated by Yomitan.
 
+The proxy is very lightweight, so it's fine to leave it running in the background. On Windows, `RBTray` can be used to minimise it to the taskbar.
```
Comment (killergerbah) [inline README.md]: Link to RBTray? For convenience.

**Review by killergerbah — CHANGES_REQUESTED** [review]

**Comment by killergerbah** [comment]
> I've also thought about git-adding the binary file for the go proxy in the repo. 

This is a nice idea. Users would not have to download Go.

>But the disadvantage is that these should be regenerated if the script contents changes, and it's difficult to provide these for all 3 OSes (not sure if you'd want PRs for these as it's kinda a security risk?).

If we can automate the building of all binaries using a script tracked in this repo I think that alleviates some security risk. I would go farther to say that the binaries should not be checked in until we've found a way to automated the build process.

**Comment by killergerbah** [comment]
Thanks for your changes. I have a final nitpick: "e.g." should be used when listing examples of things, not "i.e.

**Comment by artjomsR** [comment]
I've also thought so but originally decided to stick to i.e. for consistency :) Updated them now. Also probably a good idea to squash all these commits when you merge this PR

**Comment by killergerbah** [comment]
Okay awesome. Thanks again for all your contributions.

**Pull request merged**